tests/vxlan/vnet_utils.py

```diff
@@ -0,0 +1,166 @@
+import json
+import logging
+
+from jinja2 import Template
+from os import path
+from time import sleep
+from vnet_constants import *
+from vnet_constants import VXLAN_PORT, VXLAN_MAC
+
+logger = logging.getLogger(__name__)
+
+def safe_open_template(template_path):
+    """
+    Wraps calls to `open()` using `with` to ensure cleanup
```

Comment:
I would probably summarize this as "load a Jinja2 template from the given file" or something similar. Ideally the docstring should describe what the method does/how to use it rather than how it works.

Reply:
Fixed in next push